public/data/food/processes.json

```diff
@@ -30,9 +30,8 @@
       "tre": 0,
       "wtu": 0
     },
-    "name": "Egg, Bleu Blanc Coeur, outdoor system, at farm gate {FR} U",
     "source": "Agribalyse 3.1.1",
-    "sourceId": "AGRIBALU000000003105149",
+    "sourceId": "Egg, Bleu Blanc Coeur, outdoor system, at farm gate {FR} U",
```

Comment:
This change breaks the the link to the default process in `ingredients.json` with the `default` field.